lib/mapper.js

```diff
@@ -102,3 +77,77 @@ exports.getConversionLabel = function(eventName, events) {
     return events[eventName];
   }
 };
+
+/**
+ * You can whitelist Segment events to be sent as standalone remarketing tags
+ * Exposing this with exports because we need to use it in lib/index.js's ensure block
+ *
+ * @param {String} eventName
+ * @param {Array} events
+ * @return {Boolean}
+ */
+
+exports.getRemarketingEvent = function(eventName, events) {
+  if (events.indexOf(eventName) > -1) return true;
```

Comment:
this can just be `return events.indexOf(eventName) > -1` (this also has the benefit of being`true/false` instead of `true/undefined`)